src/components/Toasts/RegisterToast.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import Toast from "react-bootstrap/Toast";
+
+
+const getStyle = (props) => {
+  let baseClass = "bg-";
+  // the message we get from the server, if it's an error
+  if (props.message.msgBody === "Email is already taken")
```

Comment:
Change to 
`if (props.message.msgErr) message = "Sorry!"`

In case we were to change the msgBody on the backend repo

Reply:
no need